### PR TITLE
add mongodb OP_MSG (2013)

### LINKED ISSUE #6191: There is a problem when using MongoDB with packetbeat.
Please post all questions and issues on https://discuss.elastic.co/c/beats
before opening a Github Issue. Your questions will reach a wider audience there,
and if we confirm that there is a bug, then you can open a new issue.

For security vulnerabilities please only send reports to security@elastic.co.
See https://www.elastic.co/community/security for more information.

For confirmed bugs, please report:
- Version: packetbeat-6.1.2-linux-x86_64
- Operating System: Linux contents-mongo-dev-01 4.9.20-11.31.amzn1.x86_64 #1 SMP Thu Apr 13 01:53:57 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux
- Mongodb Version: db version v3.2.13
- Steps to Reproduce: 

[root@contents-mongo-dev-01 packetbeat-6.1.2-linux-x86_64]#  /beat/packetbeat-6.1.2-linux-x86_64/packetbeat -c /beat/packetbeat-6.1.2-linux-x86_64/packetbeat.yml -e 

Q) When packetbeat is executed, the following error occurs.

2018/01/26 06:30:01.311175 mongodb_parser.go:42: ERR Unknown operation code: 
2018/01/26 06:30:01.311872 mongodb_parser.go:42: ERR Unknown operation code: 
2018/01/26 06:30:01.311902 mongodb_parser.go:42: ERR Unknown operation code: 
2018/01/26 06:30:01.311928 mongodb_parser.go:42: ERR Unknown operation code: 
2018/01/26 06:30:01.403019 mongodb_parser.go:42: ERR Unknown operation code: 
2018/01/26 06:30:01.403631 mongodb_parser.go:42: ERR Unknown operation code: 

Q) My setup is as below and I simply linked it to the logstash. Please check the cause of the above error.

[root@contents-mongo-dev-01 packetbeat-6.1.2-linux-x86_64]# cat packetbeat.yml | grep -v '#'

packetbeat.interfaces.device: any

packetbeat.flows:
  timeout: 30s
  period: 10s

packetbeat.protocols:
- type: mongodb
  ports: [27017]
  send_request: true
  send_response: true
  max_docs: 0
  max_doc_length: 0

setup.template.settings:
  index.number_of_shards: 3
setup.kibana:

output.elasticsearch:
  hosts: ["internal-bh-elasticsearch-lb-1602908268.ap-northeast-2.elb.amazonaws.com:9200"]

logging.level: error

### PR DESCRIPTION
**Pull request by pohzipohzi**
this fixes #6191

### CODE REVIEW

**Comment by elasticcla** [comment]
Hi @pohzipohzi, we have found your signature in our records, but it seems like you have signed with a different e-mail than the one used in yout Git [commit](https://github.com/elastic/beats/pull/8594.patch). Can you please add both of these e-mails into your Github profile (they can be hidden), so we can match your e-mails to your Github profile?

**Comment by elasticcla** [comment]
Hi @pohzipohzi, we have found your signature in our records, but it seems like you have signed with a different e-mail than the one used in yout Git [commit](https://github.com/elastic/beats/pull/8594.patch). Can you please add both of these e-mails into your Github profile (they can be hidden), so we can match your e-mails to your Github profile?

**Comment by pohzipohzi** [comment]
@adriansr @cwurm any update on this?

**Comment by adriansr** [comment]
It looks good. However, the issue is the same as with #6440, we would like to have some basic tests for this.

Something like loading a PCAP with the new opcodes and checking that the emitted events are fine. There's a lot of examples in packetbeat, like this one https://github.com/elastic/beats/blob/master/packetbeat/tests/system/test_0001_mysql_spaces.py.

**Comment by adriansr** [comment]
@pohzipohzi will you by able to add that test?

File: CHANGELOG.asciidoc [hunk 1]
```diff
@@ -106,7 +106,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 
 *Packetbeat*
 
-- Add support for mongodb opcode 2013 (OP_MSG). {issue}6191[6191] {pull}8594[8594]
```
Comment (adriansr) [inline CHANGELOG.asciidoc]: You've lost your changes to CHANGELOG.asciidoc while merging.

Note that now we use `CHANGELOG.next.asciidoc` for unreleased functionality, so add it there.

**Comment by pohzipohzi** [comment]
@adriansr sorry for taking so long to get back to you. Honestly I haven't touched this project in some time and I'm having some trouble setting up my environment to run python related tests. I will be a little busy this week but I do intend to add the test by next week or so

**Comment by pohzipohzi** [comment]
Closing and continuing in #11500

**Pull request closed without merging**